config.yaml

```diff
@@ -12,11 +12,23 @@ listen: :8080
 # Defaults to: prod
 # middleware: dev
 
+# Private key type - used for JWT signing
+# Defaults to: rsa
+# Supported values: rsa, ed25519
+# Overwrite with environment variable: USERADM_SERVER_PRIV_KEY_TYPE
+# server_priv_key_type: rsa
```

Comment:
Same as for the deviceauth PR, this option is redundant and could be resolved by proper parsing of the PKCS#8 structure.

Reply:
done